xdp-loader/xdp-loader.8

```diff
@@ -160,6 +161,44 @@ Enable debug logging. Specify twice for even more verbosity.
 .PP
 Display a summary of the available options
 
+.SH "The FEATURES command"
+.PP
+The \fIfeatures\fP command displays the XDP features supported by the NIC.
+
+.PP
+Currently supported XDP features are:
+
+.SS "NETDEV_XDP_ACT_BASIC"
+.PP
+XDP feautues set supported by all drivers (XDP_ABORTED, XDP_DROP, XDP_PASS,
+XDP_TX).
+
+.SS "NETDEV_XDP_ACT_REDIRECT"
+.PP
+The netdev supports XDP_REDIRECT.
```

Comment:
"The network device supports handling the XDP_REDIRECT return code. This means packets can be redirected *from* this device by XDP."